src/components/Header/index.js

```diff
@@ -27,38 +27,38 @@ class Header extends Component {
       <div className="headerContainer">
         <div className="container">
           <div className="headerContainer__group headerContainer__group--logo">
-            <Link to="/">
+            <NavLink to="/">
               <div className="headerLogo beta" />
-            </Link>
+            </NavLink>
           </div>
           <div className="headerContainer__group headerContainer__group--left headerContainer__group--version">
             {version}
           </div>
           <div className="headerContainer__group headerContainer__group--left navLinks">
             {hasWallet && (
-              <Link
+              <NavLink
```

Comment:
Why the ```NavLink```?

Reply:
@apanizo We're using `activeClassName` prop and it's not on `Link` anymore.

><NavLink> works the same as <Link> but gives you some extra styling abilities depending >on if the <NavLink> matches the browser's URL.
https://css-tricks.com/react-router-4/